### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by AshutoshM29**

### CODE REVIEW

File: Pages/Modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.modal {
+    background: var(--gray-lighter);
+    border-radius: 1rem;
+    min-width: 35rem;
+    max-width: 45rem;
+    z-index: 2;
+  }
```
Comment (rahulk31) [inline Pages/Modal/modal.css:7]: Modal should have a fixed position on the viewport.

**Review by rahulk31 — COMMENTED** [review]
Modal should have a fixed position on the viewport. It should block the content of the overall page for the instance when modal is active.

The card for modal is fine. Need to work on positioning.

File: Pages/Modal/modal.css (hunk 1 above)
Comment (AshutoshM29) [reply Pages/Modal/modal.css:7]: Do I need to fix the position now, cuz I think it will work with JS

Comment (rahulk31) [reply Pages/Modal/modal.css:7]: Okay then you can plan it later. For now hopefully it should be fine 🤞

**Review by rahulk31 — APPROVED** [review]

**Comment by Naman-Saxena1** [comment]
Looks good to me 👍 
You can start adding screenshots of the changes you made in PR Request description, it was recommended by Vikram and MAs.

**Review by varun1233 — COMMENTED** [review]
Code Looks good
Just one feedback. Your close button gets out of flow when i reduce the size of the modal.
Attached the screenshot in the comments

**Pull request merged**